src/AuthModule/components/ChangPass/ChangPass.tsx

```diff
@@ -0,0 +1,195 @@
+
+              <div className="form-group conInput  my-4">
+                <label className=" color-text" htmlFor="New ">
+                  New Password
+                </label>
+                <input
+                  className={`${styleChangPass.styleInput} form-control mt-2 `}
+                  placeholder="Enter your New Password"
+                  type={showNewPassword ? "text" : "password"}
+                  {...register("newPassword", {
+                    required: "new Password is required",
+                    pattern: {
+                      value:
+                        /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/,
+                      message:
+                        "Password must contain at least 8 characters, including upper and lowercase letters, and numbers",
+                    },
+                  })}
+                  id="New"
+                />
+                <i
+                  className={`fa-regular fa-eye${
+                    showNewPassword ? "-slash" : ""
+                  }`}
+                  onClick={toggleNewPassword}
+                ></i>
+
+                {errors.newPassword && (
+                  <div className="alert alert-danger  d-inline-block w-100 mt-1">
+                    {errors.newPassword.message}
+                  </div>
+                )}
+                <div className="border_bottom"></div>
+              </div>
+
+              {/* confirmPassword */}
+
+              <div className="form-group conInput  my-4">
+                <label className="color-text" htmlFor="ConfirmPass ">
+                  Confirm New Password
+                </label>
+                <input
+                  className={`${styleChangPass.styleInput} form-control mt-2 `}
+                  type={showPasswordConfirm ? "text" : "password"}
+                  placeholder="Confirm New Password"
+                  {...register("confirmNewPassword", {
+                    required: "confirm Password is required",
+                    validate: validatePasswordMatch,
+                  })}
+                  id="ConfirmPass"
+                />
+                <i
+                  className={`fa-regular fa-eye${
+                    showPasswordConfirm ? "-slash" : ""
+                  }`}
+                  onClick={togglePasswordVisibilityconfirm}
+                ></i>
+
+                {errors.confirmNewPassword && (
+                  <div className="alert alert-danger  d-inline-block w-100 mt-1 ">
+                    {errors.confirmNewPassword.message}
+                  </div>
+                )}
+                <div className="border_bottom"></div>
+              </div>
+              <button
+                type="submit"
+                className="w-100 btn color-button rounded-5"
+              >
+                {spinner ? (
+                  <div className="spinner-border" role="status"></div>
+                ) : (
+                  "Save"
```

Comment:
We can change this text to something more meaningful such as "Save Changes" or "Change Password", because "Save" only is a little bit vague.